ddl-scripts/create_tables_yugabyte.sql

```diff
@@ -22,7 +22,7 @@ CREATE TABLE IF NOT EXISTS event_journal(
 
 -- `event_journal_slice_idx` is only needed if the slice based queries are used
 CREATE INDEX IF NOT EXISTS event_journal_slice_idx ON event_journal(slice ASC, entity_type ASC, db_timestamp ASC)
-  INCLUDE (persistence_id, seq_nr, deleted);
```

Comment:
Seems like it's not possible to combine INCLUDE and SPLIT AT, but I don't think we benefit much from the INCLUDE anyway. One thing we can watch out for is the `deleted = false` condition.

```
ERROR:  Illegal state: Number of split row values must be equal to number of primary key columns: 4 vs. 3
```

Reply:
the original reason for using the INCLUDE was for the backtracking queries that don't fetch the event payload

I created issue for follow up https://github.com/akka/akka-persistence-r2dbc/issues/184

Reply:
How does this work when there are three keys in the index, the docs only show a sample of a single primary key not using the full value-space. Is splitting at the hash value for all three combined somehow?

Reply:
`SPLIT AT VALUES` is used for range based pk/idx, i.e. when using `ASC` not `HASH`. Here we split it on the first column `slice`. There is some syntax to be able to split it on combined column values, see https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_table/#split-at-values

Reply:
Ah, thx I looked at the wrong docs section, so it can actually be a tuple up to the number of key parts. Got it!

Reply:
yes, it's not really documented for index, but seems to work the same way as for tables, which makes sense since an index is pretty much the same as an additional table